modules/io/common/src/main/java/org/locationtech/jts/io/geojson/GeoJsonConstants.java

```diff
@@ -32,5 +35,7 @@
   public static final String NAME_MULTIPOLYGON = "MultiPolygon";
   public static final String NAME_MULTILINESTRING = "MultiLineString";
   public static final String NAME_MULTIPOINT = "MultiPoint";
+  public static final String NAME_FEATURE = "Feature";
```

Comment:
Would be nice to improve the ordering of these constants.  Either alphabetically, or put the new ones required for Features at the end